demos/python_demos/colorization_demo/colorization_demo.py

```diff
@@ -98,7 +98,16 @@ def build_arg():
         img_l_rs = cv.resize(img_lab.copy(), (w_in, h_in))[:, :, 0]
 
         log.debug("Network inference")
-        res = exec_net.infer(inputs={input_blob: [img_l_rs]})
+
+        if len(load_net.input_info.keys()) == 1:
+            res = exec_net.infer(inputs={input_blob: [img_l_rs]})
+        else:
+            input_blob, user_ab_input, user_map = load_net.input_info.keys()
+            res = exec_net.infer(inputs={
+                input_blob: [img_l_rs],
+                user_ab_input: np.zeros((2, *img_l_rs.shape)),
+                user_map: [np.zeros(img_l_rs.shape)]
+            })
```

Comment:
could you please make consistent inputs filling format (everywhere np.array or everywhere starts with list)?

Reply:
Also if demo supports several input files processing, then better to create feed_dict with initiall input data and then chenge only input_blob content

Reply:
Done